week-1/exercises/decimal.js

```diff
@@ -1,9 +1,15 @@
 const num = 58.4567;
+const wholeNumberPart = Math.round(num);
```

Comment:
you should use `Math.floor` here. If `num = 58.66` then this would evaluate to 59, which would be incorrect

Reply:
Thank you.
I have changed it to Math. floor. I did not think of it as a different number when I assigned it Math. round. Thank you for making me think in that angle.